conda_mirror/diff_tar.py

```diff
@@ -0,0 +1,191 @@
+import os
+import sys
+import json
+import hashlib
+import tarfile
+from os.path import abspath, isdir, join, relpath
+
+
+mirror_dir = None
+reference_path = './reference.json'
```

Comment:
global constants should be UPPERCASE :wink:

Reply:
Just did